symplyphysics/laws/electricity/circuits/resistance_of_the_serial_resistor_coil_capacitor_circuit.py

```diff
@@ -0,0 +1,69 @@
+from sympy import (Eq, solve, sqrt, I)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output,)
+from symplyphysics.core.expr_comparisons import expr_equals
+
+from symplyphysics.definitions import impedance_is_resistance_and_reactance as impedance_law
+from symplyphysics.laws.electricity.circuits import resistivity_of_serial_resistors as serial_law
+
+# Description
+## Consider an electrical circuit consisting of a capacitor, coil, and resistor connected in series.
+## Then you can find the total resistance of such a circuit.
+
+## Law is: R = sqrt(R0^2 + (Xl - Xc)^2), where
+## R - resistance of circuit,
+## R0 - resistance of resistor
+## Xl - inductive reactance,
+## Xc - capacitive reactance.
+
+circuit_resistance = Symbol("circuit_resistance", units.impedance, real=True)
+
+resistance_resistor = Symbol("resistance_resistor", units.impedance, real=True)
+capacitive_reactance = Symbol("capacitive_reactance", units.impedance, real=True)
+inductive_reactance = Symbol("inductive_reactance", units.impedance, real=True)
+
+law = Eq(circuit_resistance, sqrt(resistance_resistor**2 + (inductive_reactance - capacitive_reactance)**2))
+
+# This law might be derived via impedance definition and "resistivity_of_serial_resistors" law.
+
+impedance_law_applied_1 = impedance_law.definition.subs({
+impedance_law.impedance: -I * capacitive_reactance,
```

Comment:
Not justified. You should use some existing law to prove it. I suggest to add law for capacitor reactance.